<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrimaryWindow</class>
 <widget class="QMainWindow" name="PrimaryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>742</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
    <pointsize>-1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>QADQScope</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: Roboto; font-size: 14px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>4</verstretch>
       </sizepolicy>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scopeTab">
       <attribute name="title">
        <string>Scope</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="ScopeTab" name="scopeTabWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="spectroscopeTab">
       <attribute name="title">
        <string>Spectrum</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="SpectroscopeTab" name="spectroscopeTabWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsTab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="PrimaryControls" name="primaryControls" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1057</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionCalibration"/>
    <addaction name="separator"/>
    <addaction name="actionDMA_Buffers"/>
    <addaction name="separator"/>
    <addaction name="actionSpectrumAnalyzer"/>
   </widget>
   <addaction name="menuConfiguration"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionCalibration">
   <property name="text">
    <string>Calibration</string>
   </property>
  </action>
  <action name="actionDMA_Buffers">
   <property name="text">
    <string>DMA Buffers</string>
   </property>
  </action>
  <action name="actionUser_logic">
   <property name="text">
    <string>User logic</string>
   </property>
  </action>
  <action name="actionSpectrumAnalyzer">
   <property name="text">
    <string>Spectrum analyzer</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScopeTab</class>
   <extends>QWidget</extends>
   <header>ScopeTab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpectroscopeTab</class>
   <extends>QWidget</extends>
   <header>SpectroscopeTab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PrimaryControls</class>
   <extends>QWidget</extends>
   <header>PrimaryControls.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
